book/guide/tuning.md

```diff
@@ -1,81 +1,235 @@
-# Tuning
+# Performance Tuning
+
+## Overview
+The Firedancer validator is composed of a handful of threads, each
+performing one of eleven distinct jobs. Some jobs only need one thread
+to do them, but certain jobs require many threads performing the same
+work in parallel.
+
+Each thread is given a CPU core to run on, and threads take ownership of
+the core: never sleeping or letting the operating system use it for
+another purpose. The combination of a job, and the thread it runs on,
+and the CPU core it is assigned to is called a tile. The eleven kinds of
+tile are,
+
+| Tile   | Description |
+|--------|-------------|
+| `net`  | Responsible for sending and receiving network packets from the network device |
+| `quic` | A QUIC server for receiving transactions from clients. Performs all connection management and packet processing to manage and implement the QUIC protocol |
```

Comment:
This is the only one that doesn't start with a verb.
Maybe "Receives transactions from clients, performing all ..."